src/MCPClient/lib/archivematicaClient.py

```diff
@@ -143,6 +143,29 @@ def set_start_times():
     return jobs
 
 
+def _parse_command_line(s):
+    return [_shlex_unescape(x) for x in shlex.split(s)]
+
+
+def _shlex_unescape(s):
+    result = ""
+
+    i = 0
+    while i < len(s):
+        if (i + 1) < len(s) and s[i] == '\\' and s[i + 1] == '`':
+            # If we're looking at an escaped backtick, drop the escape
+            # character.  Shlex doesn't do this but bash unescaping does, and we
+            # want to remain compatible.
+            result += s[i + 1]
+            i += 1
+        else:
+            result += s[i]
+
+        i += 1
+
+    return result
```

Comment:
@marktriggs this func could be replaced with something more concise like the following, though depending on your taste it may reduce readability (in which case feel free to ignore):

    def _shlex_unescape(s):
        return ''.join(c1 for c1, c2 in zip(a, a[1:] + '.') if (c1, c2) != ('\\', '`'))

Reply:
Nice!  I'll take it :)  I've just updated the PR branch, thanks.